### ISSUE TITLE
Anorm 2.6.3 depends on scala-arm 2.1-SNAPSHOT, which does not exist

### ISSUE THREAD

**Issue by btmcnellis** [playframework/anorm#208]
We recently attempted to build one of our projects against the new Anorm 2.6.3 and found that we were unable to resolve one of its dependencies:

```
[error] coursier.lmcoursier.ResolutionException: Encountered 1 error(s) in dependency resolution:
[error]     com.jsuereth:scala-arm_2.12:2.1-SNAPSHOT:
```

It appears that there is no scala-arm 2.1-SNAPSHOT release. On further investigation, it appears that a fork was published to this repository: https://github.com/cchantep/tatami

We could add a resolver for that repository to our build.sbt, but for now we're probably just going to stick with Anorm 2.6.2, since it frankly feels kind of weird to depend on some undocumented fork of an otherwise unmaintained library. Is there a plan to remove the scala-arm dependency at some point, or at least to fork it properly and open source any of the changes?

Also, the README and other documentation should be updated to reflect that this new resolver is needed. Otherwise people are going to have trouble resolving this.

**Comment by cchantep** [comment]
Anorm 2.6.4 is shading Scala-ARM, until Scala-ARM 2.1 is release (as it's not unmaintained).

**Comment by EwanValentine** [comment]
I'm new to Scala, I'm trying to use Anorm with Play. I'm using Scala 2.13, trying to install Anorm with:
```
libraryDependencies ++= Seq(
  jdbc,
  "org.playframework.anorm" %% "anorm" % "2.6.3"
)
```

But I'm getting this error:
```
[error] sbt.librarymanagement.ResolveException: unresolved dependency: com.jsuereth#scala-arm_2.13;2.1-SNAPSHOT: not found
```

Apologies if this is a stupid question!

**Comment by platypii** [comment]
Fixed in anorm version 2.6.4, try upgrading.

**Comment by davidjohnmaccallum** [comment]
The "Add Anorm to your project" section of the Anorm home page (https://playframework.github.io/anorm/) needs to be updated. It is referencing 2.6.3
